apps/src/sites/studio/pages/schoolInfoConfirmationDialog.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file Renders the SchoolInfoConfirmationDialog component on page load.
+ * This file is responsibile for mounting and unmounting the React component,
+ * and providing props passed down from the server to the component.
+ * @see _school_info_confirmation_dialog.html.haml.
+ */
```

Comment:
:+1: I love seeing clear entry point comments like this that provide needed context!

Reply:
Thanks!